app/src/main/java/com/worldsnas/starplayer/model/Music.kt

```diff
@@ -1,10 +1,14 @@
 package com.worldsnas.starplayer.model
 
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
```

Comment:
iin laye payini hast. nabayad az android khabar dashte bashe

Reply:
mn yekam gij shodam, alan Music bere be laye haye balatar dige!
Response va LocalMusic har 2 miad dakhele repo, Music mire be ViewModel, in nazare mane(be in shekl taqir bedam)

Reply:
na nakon inkaro
bar aksse daghighan
ba intafavot ke `LocalMusic` faghat baraye pass dadan be fragment dige estefade mishe na chize dige
`Music` estelahan contract e ma hast baraye laye haye "model" mesle Repository ya ViewModel